recipes/_lustre_install.rb

```diff
@@ -68,12 +68,15 @@
   end
 
   kernel_module 'lnet'
+
 elsif node['platform'] == 'centos'
+  # Centos 6
   Chef::Log.warn("Unsupported version of Centos, #{node['platform_version']}, supported versions are >= 7.5")
+
 elsif node['platform'] == 'ubuntu'
 
   apt_repository 'fsxlustreclientrepo' do
-    uri          node['cfncluster']['lustre']['repository_uri']
+    uri          node['cfncluster']['lustre']['base_url']
```

Comment:
wasn't this the actual repository_uri? why changing to base_url?

Reply:
> Renamed ubuntu specific repository_uri attribute to be aligned with other OSes.

`apt_repository / uri` has the same meaning of `yum_repository / baseurl` and I wanted to avoid using a different variable for this os only and be able to initialize it with a single `value_for_platform` assignment, instead of having an if/else block on which the variable changes name.